src/diffusers/pipelines/flux2/system_messages.py

```diff
@@ -0,0 +1,35 @@
+"""
+These system prompts come from:
+https://github.com/black-forest-labs/flux2/blob/5a5d316b1b42f6b59a8c9194b77c8256be848432/src/flux2/system_messages.py#L54
+"""
+
+SYSTEM_MESSAGE = """You are an AI that reasons about image descriptions. You give structured responses focusing on object relationships,
```

Comment:
As discussed internally, this new-line character thingy messes the quality a bit. Hence, I have decided to keep these system messages one-to-one same as the original implementation linked above. 

If we run `make style && make quality`, this order will be completely destroyed. We can change the `pyproject.toml` to exclude this path from getting formatted. But before we do that let's see if this is the best we have.